cli/tests/test_help_command.rs

```diff
@@ -84,3 +79,145 @@ fn test_help() {
+              useful e.g. in a command prompt, especially if you have another process that commits the
+              working copy.
+              
+              Loading the repository at a specific operation with `--at-operation` implies
+              `--ignore-working-copy`.
+
+          --ignore-immutable
+              Allow rewriting immutable commits
+              
+              By default, Jujutsu prevents rewriting commits in the configured set of immutable commits.
+              This option disables that check and lets you rewrite any commit but the root commit.
+              
+              This option only affects the check. It does not affect the `immutable_heads()` revset or
+              the `immutable` template keyword.
+
+          --at-operation <AT_OPERATION>
+              Operation to load the repo at
+              
+              Operation to load the repo at. By default, Jujutsu loads the repo at the most recent
+              operation, or at the merge of the divergent operations if any.
+              
+              You can use `--at-op=<operation ID>` to see what the repo looked like at an earlier
+              operation. For example `jj --at-op=<operation ID> st` will show you what `jj st` would
+              have shown you when the given operation had just finished. `--at-op=@` is pretty much the
+              same as the default except that divergent operations will never be merged.
+              
+              Use `jj op log` to find the operation ID you want. Any unambiguous prefix of the operation
+              ID is enough.
+              
+              When loading the repo at an earlier operation, the working copy will be ignored, as if
+              `--ignore-working-copy` had been specified.
+              
+              It is possible to run mutating commands when loading the repo at an earlier operation.
+              Doing that is equivalent to having run concurrent commands starting at the earlier
+              operation. There's rarely a reason to do that, but it is possible.
+              
+              [aliases: at-op]
+
+          --debug
+              Enable debug logging
+
+          --color <WHEN>
+              When to colorize output (always, never, debug, auto)
+
+          --quiet
+              Silence non-primary command output
+              
+              For example, `jj file list` will still list files, but it won't tell you if the working
+              copy was snapshotted or if descendants were rebased.
+              
+              Warnings and errors will still be printed.
+
+          --no-pager
+              Disable the pager
+
+          --config-toml <TOML>
+              Additional configuration options (can be repeated)
+
+      -h, --help
+              Print help (see a summary with '-h')
+
+      -V, --version
+              Print version
+    "#);
+
+    // Help command should work category commands
```

Comment:
The comment "Help command should work category commands"
doesn't make sense.

Reply:
Ops, missed a "with"